app/templates/admin/destinations/list.html

```diff
@@ -28,6 +29,7 @@ <h1>Your Destinations</h1>
             <td><a href="{{ url_for('admin.destinations_show', uuid=dest.uuid) }}">{{ dest.name }}</a></td>
             <td>{{ dest.address }}</td>
             <td>{{ dest.hidden }}</td>
+            <td><a href="{{ url_for('destination.page', uuid=dest.uuid) }}">{{ request.host_url[0:-1]  + url_for('destination.page', uuid=dest.uuid) }}</a></td>
```

Comment:
You can use `external=True` as an arg to `url_for()` to get the fully qualified URL.